org.eclipse.xtext.web/src/main/js/xtext/services/XtextService.js

```diff
@@ -164,7 +164,11 @@ define(['jquery'], function(jQuery) {
 	XtextService.prototype.sendRequest = function(editorContext, settings, needsSession) {
 		var self = this;
 		self.setState('started');
-		
+		var corsEnabled = editorContext._editor.options['enableCors'];
+		if(corsEnabled !== undefined && corsEnabled) {
```

Comment:
Redundant condition: just write `if (corsEnabled)` (`undefined` is evaluated to `false`)

Reply:
Thanks! I thought that was the case. I will make that change.